### DISCUSSION TITLE
Getting unhandled exception in .net saying parameter is null.

### DISCUSSION THREAD

**Discussion by Hoomum** [audiamus/BookLibConnect#177] [Q&A]
I am getting the unhandled exception when starting up the app and loading databases. I have tried reinstalling both apps but not getting anywhere. Any ideas how I can fix this please?
![Screenshot 2024-12-05 131605](https://github.com/user-attachments/assets/5c3e1173-02e3-48ed-b9ab-1b56c340a958)

**Comment by laurelwreath-az** [comment]
I have the same problem here is the details. Hope someone who knows more than me can help.
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentNullException: Value cannot be null. (Parameter 'input')
   at System.Text.RegularExpressions.ThrowHelper.ThrowArgumentNullException(ExceptionArgument arg)
   at System.Text.RegularExpressions.Regex.Match(String input)
   at core.audiamus.connect.BookLibrary.addSeries(Book book, ICollection`1 series, ICollection`1 seriesBooks, IEnumerable`1 itmRelations)
   at core.audiamus.connect.BookLibrary.addPageBooks(BookDbContextLazyLoad dbContext, BookCompositeLists bcl, IEnumerable`1 products, ProfileId profileId, Boolean resync)
   at core.audiamus.connect.BookLibrary.addRemBooks(List`1 libProducts, ProfileId profileId, Boolean resync)
   at core.audiamus.connect.BookLibrary.<>c__DisplayClass7_0.<AddRemBooksAsync>b__1()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at core.audiamus.connect.BookLibrary.AddRemBooksAsync(List`1 libProducts, ProfileId profileId, Boolean resync)
   at core.audiamus.connect.AudibleApi.GetLibraryAsync(String json, Boolean resync)
   at core.audiamus.connect.AudibleApi.GetLibraryAsync(Boolean resync)
   at core.audiamus.connect.app.gui.MainForm.initLibraryAsync()
   at core.audiamus.connect.app.gui.MainForm.init()
   at core.audiamus.connect.app.gui.MainForm.OnShown(EventArgs e)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)


************** Loaded Assemblies **************
System.Private.CoreLib
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Private.CoreLib.dll
----------------------------------------
BookLibConnect
    Assembly Version: 0.13.1.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Runtime
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.3624.51513
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/System.Windows.Forms.dll
----------------------------------------
System.Windows.Forms.Primitives
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.3624.51513
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/System.Windows.Forms.Primitives.dll
----------------------------------------
audiamus.aux.core
    Assembly Version: 2.2.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.ComponentModel.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ComponentModel.Primitives.dll
----------------------------------------
System.Runtime.InteropServices
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.InteropServices.dll
----------------------------------------
System.Drawing.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Drawing.Primitives.dll
----------------------------------------
System.Collections.Specialized
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Collections.Specialized.dll
----------------------------------------
System.ComponentModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ComponentModel.dll
----------------------------------------
System.Drawing.Common
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/System.Drawing.Common.dll
----------------------------------------
System.Threading
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Threading.dll
----------------------------------------
System.Diagnostics.TraceSource
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Diagnostics.TraceSource.dll
----------------------------------------
System.Collections
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Collections.dll
----------------------------------------
System.Text.RegularExpressions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Text.RegularExpressions.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.InteropServices.RuntimeInformation.dll
----------------------------------------
System.Memory
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Memory.dll
----------------------------------------
System.Text.Json
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Text.Json.dll
----------------------------------------
System.Text.Encodings.Web
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Text.Encodings.Web.dll
----------------------------------------
System.Collections.Concurrent
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Collections.Concurrent.dll
----------------------------------------
System.Runtime.Intrinsics
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.Intrinsics.dll
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Numerics.Vectors.dll
----------------------------------------
System.Private.Uri
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Private.Uri.dll
----------------------------------------
System.Reflection.Emit.Lightweight
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Emit.Lightweight.dll
----------------------------------------
System.Reflection.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Primitives.dll
----------------------------------------
System.Reflection.Emit.ILGeneration
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Emit.ILGeneration.dll
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.CompilerServices.Unsafe.dll
----------------------------------------
System.Threading.Thread
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Threading.Thread.dll
----------------------------------------
System.Linq
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Linq.dll
----------------------------------------
System.Console
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Console.dll
----------------------------------------
System.Text.Encoding.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Text.Encoding.Extensions.dll
----------------------------------------
audiamus.aux.win.core
    Assembly Version: 2.2.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
audiamus.common.util.core
    Assembly Version: 0.3.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
audiamus.connect.core
    Assembly Version: 0.13.1.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
audiamus.booksdb.core
    Assembly Version: 0.6.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
audiamus.connect.ui.core
    Assembly Version: 0.13.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Win32.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/Microsoft.Win32.Primitives.dll
----------------------------------------
System.ComponentModel.EventBasedAsync
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ComponentModel.EventBasedAsync.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.3624.51513
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/Accessibility.dll
----------------------------------------
System.ComponentModel.TypeConverter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ComponentModel.TypeConverter.dll
----------------------------------------
audiamus.common.types.core
    Assembly Version: 0.3.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Runtime.Loader
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.Loader.dll
----------------------------------------
Microsoft.Win32.SystemEvents
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/Microsoft.Win32.SystemEvents.dll
----------------------------------------
audiamus.aux.propgrid.core
    Assembly Version: 0.2.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Resources.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/System.Resources.Extensions.dll
----------------------------------------
System.Drawing
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.3624.51513
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.36/System.Drawing.dll
----------------------------------------
System.ObjectModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ObjectModel.dll
----------------------------------------
System.Collections.NonGeneric
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Collections.NonGeneric.dll
----------------------------------------
System.Threading.ThreadPool
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Threading.ThreadPool.dll
----------------------------------------
audiamus.aux.treedecomp.core
    Assembly Version: 2.1.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Net.Http
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.Http.dll
----------------------------------------
System.Net.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.Primitives.dll
----------------------------------------
System.Diagnostics.Tracing
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Diagnostics.Tracing.dll
----------------------------------------
System.Diagnostics.DiagnosticSource
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Diagnostics.DiagnosticSource.dll
----------------------------------------
System.Net.Security
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.Security.dll
----------------------------------------
System.Security.Cryptography.X509Certificates
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Cryptography.X509Certificates.dll
----------------------------------------
System.Net.Sockets
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.Sockets.dll
----------------------------------------
Microsoft.EntityFrameworkCore
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Threading.Overlapped
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Threading.Overlapped.dll
----------------------------------------
System.Net.NameResolution
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.NameResolution.dll
----------------------------------------
audiamus.aux.win.dialogbox.core
    Assembly Version: 2.1.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
audiamus.sysmgmt.core
    Assembly Version: 1.2.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Management
    Assembly Version: 4.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Win32.Registry
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/Microsoft.Win32.Registry.dll
----------------------------------------
System.Security.Cryptography.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Cryptography.Primitives.dll
----------------------------------------
System.Security.Principal.Windows
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Principal.Windows.dll
----------------------------------------
System.Security.Claims
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Claims.dll
----------------------------------------
System.Security.Cryptography.Encoding
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Cryptography.Encoding.dll
----------------------------------------
System.Security.Cryptography.Algorithms
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Security.Cryptography.Algorithms.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Relational
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Collections.Immutable
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Collections.Immutable.dll
----------------------------------------
Microsoft.Extensions.DependencyInjection.Abstractions
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.DependencyInjection
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Logging.Abstractions
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Linq.Expressions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Linq.Expressions.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Abstractions
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Caching.Abstractions
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
netstandard
    Assembly Version: 2.1.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/netstandard.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Private.CoreLib.dll
----------------------------------------
Microsoft.EntityFrameworkCore.Sqlite
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Data.Common
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Data.Common.dll
----------------------------------------
System.Transactions.Local
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Transactions.Local.dll
----------------------------------------
Microsoft.Extensions.Logging
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Options
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Configuration.Abstractions
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Caching.Memory
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.Extensions.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.ComponentModel.Annotations
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.ComponentModel.Annotations.dll
----------------------------------------
System.Net.NetworkInformation
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Net.NetworkInformation.dll
----------------------------------------
Microsoft.Data.Sqlite
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
SQLitePCLRaw.core
    Assembly Version: 2.0.6.1341
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
SQLitePCLRaw.batteries_v2
    Assembly Version: 2.0.6.1341
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
SQLitePCLRaw.provider.e_sqlite3
    Assembly Version: 2.0.6.1341
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Microsoft.EntityFrameworkCore.Proxies
    Assembly Version: 6.0.7.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
Castle.Core
    Assembly Version: 4.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Reflection
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.dll
----------------------------------------
System.Reflection.TypeExtensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.TypeExtensions.dll
----------------------------------------
System.Globalization
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Globalization.dll
----------------------------------------
System.Runtime.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.Extensions.dll
----------------------------------------
System.Reflection.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Extensions.dll
----------------------------------------
System.Reflection.Emit
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Emit.dll
----------------------------------------
DynamicProxyGenAssembly2
    Assembly Version: 0.0.0.0
    Win32 Version: n/a
    CodeBase: file:///C:/Program%20Files/audiamus/Book%20Lib%20Connect/Castle.Core.dll
----------------------------------------
System.Diagnostics.Debug
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Diagnostics.Debug.dll
----------------------------------------
System.Runtime.Serialization.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.Serialization.Primitives.dll
----------------------------------------
DynamicProxyGenAssembly2
    Assembly Version: 0.0.0.0
    Win32 Version: n/a
    CodeBase: file:///C:/Program%20Files/audiamus/Book%20Lib%20Connect/Castle.Core.dll
----------------------------------------
System.Linq.Queryable
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Linq.Queryable.dll
----------------------------------------
audiamus.audiible.json.core
    Assembly Version: 0.5.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Xml.ReaderWriter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Xml.ReaderWriter.dll
----------------------------------------
System.Private.Xml
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Private.Xml.dll
----------------------------------------
System.Formats.Asn1
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Formats.Asn1.dll
----------------------------------------
System.Runtime.Numerics
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Runtime.Numerics.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.IO.Compression.dll
----------------------------------------
System.Diagnostics.StackTrace
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Diagnostics.StackTrace.dll
----------------------------------------
System.Reflection.Metadata
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.3624.51421
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.36/System.Reflection.Metadata.dll
----------------------------------------

************** JIT Debugging **************
